setup.py

```diff
@@ -19,15 +19,15 @@ def cythonize(*args, **kwargs):
     long_description = ""
 
 setup(
-    version="1.9",
+    version="1.10",
     description="High performance Trie and Ahocorasick automata (AC automata) for python",
     name="cyac",
     url="https://github.com/nppoly/cyac",
     author="nppoly",
     author_email="nppoly@foxmail.com",
     packages=["cyac"],
     package_dir={'cyac': 'lib/cyac'},
-    package_data={'cyac': ['*.pxd', 'cyac/unicode_portability.c']},
+    package_data={'cyac': ['*.pxd', 'cyac/unicode_portability.c', 'cyac/unicodectype_db.h']},
```

Comment:
Mistype: "unicodetype_db.h" (without "c" between "unicode" and "type").

Reply:
new version is uploaded

Reply:
if you are using this library in product env, please also do some perf, if there are any bottlenecks. I'm happy to optimize it.

Reply:
@chenkovsky thank you! First I'll try it on CPython 3.12, because not all dependencies which I'm using in production environment are compatible with CPython 3.13 now. When I could update to version 3.13 I'll give you feedback here about usage in production.